src/picomsg/src/common_c_api/common.cpp

```diff
@@ -197,6 +197,17 @@ void pm_proto_prefix_to_string_for_logs(const struct pm_proto_prefix* const prot
 	sprintf(prefix_str+strlen(prefix_str), "/%d", proto_prefix->prefix_len);
 }
 
+bool pm_proto_prefix_to_string(const struct pm_proto_prefix* const proto_prefix,
```

Comment:
Same comment as in `ts_to_prefix_string()` - This function is identical to `pm_proto_prefix_to_string_for_logs()`. So make `pm_proto_prefix_to_string_for_logs()` call this function.